### ISSUE TITLE
Can't open the "update error" notification in Android 11 as the permission was denied

### ISSUE THREAD

**Issue by asif-mistry** [tachiyomiorg/tachiyomi#5467]
### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.11.1](https://github.com/tachiyomiorg/tachiyomi/releases/tag/v0.11.1)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

### Tachiyomi version

0.11.1

### Android version

Android 11

### Device

Xiomi Redmi Note 9S

### Steps to reproduce

1. Add a comic from one of the extensions which will not update properly.
2. Enable "Show update errors notification" in General settings.
3. Update the library and put the app in background.
4. Once the update check is finished notice the update errors notification.
5. When clicking this notification it tries to open it in text editor.
6. The text editor app cant open this error file as it does not have permission.

### Expected behavior

In step 6 the update error log file should be opened to show the complete list of failed updates. 

### Actual behavior

In step 6 the update error log file is not opened as this file is stored in protected folder of Tachiyomi which is inaccessible to other apps. 

### Other details

Please store this update error log file in a folder that is accessible by other apps or include an option to see the failed updates in the app itself.

**Comment by Soitora** [comment]
/edit-title "Can't open the \"update error\" notification in Android 11 as the permission was denied"

**Comment by curche** [comment]
Can't reproduce, works as expected for me. 
Can you check if the app has permissions granted? I think this might be somehow related to this being a Xiaomi device (known to cause multiple problems)

**Comment by arkon** [comment]
What app are you trying to open it with?

**Comment by asif-mistry** [comment]
I tried opening it with multiple apps including Google keep. I also tried Xplore file manager and gave it permissions to access the Android/Data folder, even after that the update error notification did not open.

**Comment by arkon** [comment]
I'm confused, Google Keep isn't even meant to open arbitrary files like that. What are the options when you actually tap the "Open log" action in the notification?

**Comment by asif-mistry** [comment]
Sorry I checked again and Google Keep is not available in the list, other Text editing apps and Chrome was available. Xplore also had text editor and text viewer options.
None of the options worked.

**Comment by curche** [comment]
Could you also try a File Manager like [Material Files](https://play.google.com/store/apps/details?id=me.zhanghai.android.files) ?

**Comment by asif-mistry** [comment]
> Could you also try a File Manager like [Material Files](https://play.google.com/store/apps/details?id=me.zhanghai.android.files) ?

![Screenshot_2021-07-05-15-02-27-723_me.zhanghai.android.files.jpg](https://user-images.githubusercontent.com/6229654/124450451-0ef04580-dda2-11eb-8454-d7ae34bb31e5.jpg)

**Comment by kennethtran93** [comment]
Did you grant additional permissions for your apps to be able to read/use files and media? Is your app/configured storage on the internal storage or external SD card? 

Just putting my notes here that it works on my device - Samsung Galaxy S10
App version: 0.11.1 (standard, 0f3f1e9, 63, 2021-05-31T23:39Z)
Android version: 11 (SDK 30)
Android build ID: RP1A.200720.012.G973WVLU5GUE2
Device brand: samsung
Device manufacturer: samsung
Device name: beyond1q
Device model: SM-G973W
Device product name: beyond1qltecs

**Comment by arkon** [comment]
Not sure what's wrong, but this seems like an issue specific to your device.
